### PR TITLE
Add more logging, prefix release

### PR DESCRIPTION
**Pull request by fbertsch**
I can't reproduce the error we're seeing on prod locally, so adding some logging to help pin down this issue (and possible issues in the future).

### CODE REVIEW

File: probe_scraper/scrapers/moz_central_scraper.py [hunk 1]
```diff
@@ -181,6 +181,8 @@ def download_files(channel, node, temp_dir, error_cache, version, tree=None):
     if tree is None:
         uri = CHANNELS[channel]['base_uri']
     else:
+        if not tree.startswith("releases/") and tree != "mozilla-central":
```
Comment (fbertsch) [inline probe_scraper/scrapers/moz_central_scraper.py]: This issue occurred locally but not on prod - for some reason, sometimes buildhub returns `mozilla-beta` or `mozilla-release` with `releases/` prefixed, but sometimes not.

Comment (Dexterp37) [reply probe_scraper/scrapers/moz_central_scraper.py]: Woha. That's weird. Would you kindly add your comment before this line ?

File: probe_scraper/scrapers/moz_central_scraper.py [hunk 2]
```diff
@@ -329,15 +331,17 @@ def scrape_channel_revisions(folder=None, min_fx_version=None, max_fx_version=No
 
         print("\nRetreiving Buildhub results for channel " + channel)
 
-        revision_dates = bh.get_revision_dates(channel, min_fx_version, max_version=max_fx_version)
+        revision_dates = bh.get_revision_dates(channel, min_fx_version, max_version=max_fx_version,
+                                               verbose=(channel == "beta"))
```
Comment (Dexterp37) [inline probe_scraper/scrapers/moz_central_scraper.py]: Why is this specifically verbose only for beta?

Comment (fbertsch) [reply probe_scraper/scrapers/moz_central_scraper.py]: Hmm, this shouldn't have been included.

**Comment by fbertsch** [comment]
@Dexterp37 RFAL

**Pull request merged**